darts/utils/statistics.py

```diff
@@ -305,9 +305,54 @@ def plot_acf(ts: TimeSeries,
     axis.plot((0, max_lag + 1), (0, 0), color='black')
 
 
+def plot_hist(data: Union[TimeSeries, List[float], np.ndarray],
+              bins: Optional[Union[int, np.ndarray, List[float]]] = None,
+              density: bool = False,
+              title: Optional[str] = None,
+              fig_size: Optional[Tuple[int, int]] = None,
+              ax: Optional[plt.axis] = None) -> None:
+    """ This function plots the histogram of `data`.
+
+    Parameters
+    ----------
+    data
+        Univariate, determerministic TimeSeries instance representing residuals.
+    bins
+        Optionally, either an integer value for the number of bins to be displayed
+        or an array-like of floats determining the position of bins.
+    density
+        bool, if `density` is set to True, the bin counts will be converted to probability density
+    title
+        The title of the figure to be displayed
+    fig_size
+        The size of the figure to be displayed.
+    ax
+        Optionally, an axis object to plot the histogram on.
+    """
+
+    raise_if_not(any([isinstance(data, x) for x in [TimeSeries, list, np.ndarray]]),
+                 'input data must be a TimeSeries instance or an array-like')
+
+    if isinstance(data, TimeSeries):
+        data._assert_univariate()
+        data._assert_deterministic()
```

Comment:
If the series is stochastic you could imagine getting something like `values = data.all_data().flatten()`.